src/IObjectMakr.ts

```diff
@@ -63,6 +63,12 @@ export interface IObjectMakrSettings {
      * be run when made.
      */
     onMake?: string;
+
+    /**
+     * Optionally, a custom functions mapping for the classes can be passed
+     * in, otherwise a default mapping will be made
+     */
+    functions?: any;
```

Comment:
This should have its own type. `any` implies it could be a primitive or an object or a class instance.